wandb/sdk/internal/internal_api.py

```diff
@@ -994,6 +1001,81 @@ def ack_run_queue_item(self, item_id, run_id=None):
             )
         return response["ackRunQueueItem"]["success"]
 
+    @normalize_exceptions
+    def create_launch_agent(self, entity, project, queues):
```

Comment:
We'll need to add some introspection here to make sure that the server can handle these queries, and if not just fall back to launch agents without any tracking.